server2/Arcadia.Assistant/Arcadia.Assistant.Calendar.Abstractions/Arcadia.Assistant.Calendar.Abstractions.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <Platforms>AnyCPU;x64</Platforms>
```

Comment:
Is  there any reason to not use default values?

Reply:
Done

Reply:
Not done.

Reply:
Deleted now :)